templates/admin/dashboard.tmpl

```diff
@@ -76,9 +76,9 @@
 			{{ctx.Locale.Tr "admin.dashboard.system_status"}}
 		</h4>
 		{{/* TODO: make these stats work in multi-server deployments, likely needs per-server stats in DB */}}
-		<div class="no-loading-indicator tw-hidden"></div>
 		<div hx-get="{{$.Link}}/system_status" hx-swap="morph:innerHTML" hx-trigger="every 5s" hx-indicator=".no-loading-indicator" class="ui attached table segment">
 			{{template "admin/system_status" .}}
 		</div>
+		<div class="no-loading-indicator tw-hidden"></div>
```

Comment:
I think it's better to use:

```
<div class="ui attached table segment">
    <div class="no-loading-indicator tw-hidden"></div>
    <div hx-get>{{template "admin/system_status" .}}</div>
</div>
```


Then it won't cause new problems if there would be a new "segment" appended in the future.

    <div>

Reply:
Yes, it could be anywhere in the DOM, I would even prefer if we not have it in the template at all, so maybe we could even make a script that dynamically appends it to document.body when `hx-indicator=".no-loading-indicator"` is present on the page.

Reply:
Or maybe just always have this invisible div at the end of body.

Reply:
> Or maybe just always have this invisible div at the end of body.

I do not think it's worth to put such element in every page, it introduces unnecessary dependencies and pollution.

I would prefer to do https://github.com/go-gitea/gitea/pull/31363#discussion_r1639089399 at the moment.

In the future, if there are more similar requirements, we could write a htmx extension.

Reply:
Maybe it can be fixed in htmx, I need to read up on that issue again.